pkg/kubelet/cm/cpuset/cpuset.go

```diff
@@ -75,6 +75,15 @@ func NewCPUSet(cpus ...int) CPUSet {
 	return b.Result()
 }
 
+// NewCPUSet returns a new CPUSet containing the supplied elements, as slice of int64.
+func NewCPUSetInt64(cpus ...int64) CPUSet {
+	b := NewBuilder()
+	for _, c := range cpus {
+		b.Add(int(c))
+	}
+	return b.Result()
+}
+
```

Comment:
Is this used anywhere?

Reply:
yes: https://github.com/kubernetes/kubernetes/pull/95734/commits/0e5f114a1ca678d7b0a406647766e141e4147f88#diff-5cb7ca1bf56d1b07b77b5bd67b9be08c8bf81063529fc6e32fb9521cbb15b6c2R444

Reply:
Do you want me to move this code in `podresources_test.go` ?

Reply:
No. It’s fine here and could prove useful for others

Reply:
ack